yellowbrick/base.py

```diff
@@ -407,3 +408,119 @@ def predict(self, X, y):
         """
         for model in self.models:
             yield cvp(model, X, y, cv=12)
+
+
+
+#TODO it appears that the previous class (MultiModelMixin) is not actually used
+# anywhere and would be redundant to this new class. Remove that one?
+class MultipleVisualizer(Visualizer):
+    """
+    Used as a base class for visualizers that use subplots.
+
+    Parameters
+    ----------
+    visualizers : A list of instantiated visualizers
+
+    nrows: integer, default: None
+        The number of rows desired, if you would like a fixed number of rows.
+        Specify only one of nrows and ncols, the other should be None. If you 
+        specify nrows, there will be enough columns created to fit all the 
+        visualizers specified in the visualizers list. 
+    
+    ncols: integer, default: None
+        The number of columns desired, if you would like a fixed number of columns.
+        Specify only one of nrows and ncols, the other should be None. If you 
+        specify ncols, there will be enough rows created to fit all the 
+        visualizers specified in the visualizers list. 
+
+    axarr: matplotlib.axarr, default: None.
+        If you want to put the plot onto an existing axarr, specify it here. Otherwise a new
+        one will be created. 
+
+    kwargs : additional keyword arguments, default: None
+        Any additional keyword arguments will be passed on to the fit() method and therefore 
+        passed on to the fit() method of the wrapped estimators, if applicable. Otherwise ignored.
+
+    Examples
+    --------
+    >>> from yellowbrick.base import MultipleVisualizer
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> from yellowbrick.classifier import ConfusionMatrix
+    >>> from yellowbrick.classifier import ClassBalance
+    >>> model = LogisticRegression()
+    >>> visualizers = [ClassBalance(model),ConfusionMatrix(model)]
+    >>> mv = MultipleVisualizer(visualizers, ncols=2)
+    >>> mv.fit(X_train, y_train)
+    >>> mv.score(X_test, y_test)
+    >>> mv.poof()
+    """
+    def __init__(self, visualizers = [], nrows = None, ncols = None, axarr = None, **kwargs):
+        self.visualizers = visualizers
+        self.plotcount = len(visualizers)
+        if nrows == None and ncols == None:
+            self.ncols = 1
+            self.nrows = self.plotcount
```

Comment:
Perhaps we could check to see if visualizers is a two-dimensional array in case the user specifies a grid?

Reply:
That would be a good enhancement. For now I've just put a TODO in the code as a reminder and we can see how we end up using this class.